<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displayTables</class>
 <widget class="QDialog" name="displayTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="cb_table"/>
     </item>
     <item>
      <widget class="QComboBox" name="cb_companyName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_download">
       <property name="text">
        <string>Alle herunterladen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_gold">
       <property name="text">
        <string>Gold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_silver">
       <property name="text">
        <string>Silber</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_bronze">
       <property name="text">
        <string>Bronze</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_supporter">
       <property name="text">
        <string>Supporter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tv_table"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_save">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>displayTables</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>displayTables</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
